CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## v0.9.1 (2025-03-06)
+
+* Fetch `stdalign.h` from system on FreeBSD (#590, @shym, @dinosaure)
```

Comment:
```suggestion
* Fetch `stdalign.h` from system on FreeBSD (#590, @shym, @dinosaure)
* Fix the documentation about clocks (sorry for my french) by @dinosaure in #591
```

Reply:
Not really important but I can add it :+1:

Reply:
I don't have any opinion on this; it was just what was in the github release